docs/docs/policies/scopes.md

```diff
@@ -0,0 +1,72 @@
+# Scopes
+
+Scopes define the workload this policy cares about. For example,
+we want to know all dns events executed by the tool `dig`:
+
+Example 1:
+
+```
+name: dig
+description: This policy traces dns events from the dig binary
+scope:
+  - binary=/usr/bin/dig
+defaultAction: log
+rules:
+  - event: net_packet_dns
+```
+
+or 
+
+Example 2: 
+
+```
+name: dig
+description: This policy traces dns events from the dig binary
+scope:
+  - comm=dig
+defaultAction: log
+rules:
+  - event: net_packet_dns
+```
+
+To track events in the whole host, use the `global` scope:
+
+```
+name: sched_process_exec
+description: This policy traces all processes executed on the host
+scope:
+  - global
+defaultAction: log
+rules:
+  - event: sched_process_exec
+```
+
+Or, if I want all `sched_process_exec` on the host, ignoring containers:
+
+```
+name: sched_process_exec
+description: This policy traces all processes executed on the host
+scope:
+  - !container
+defaultAction: log
+rules:
+  - event: sched_process_exec
+```
+
+
+   !!! Note
```

Comment:
can you either provide a sentence about each -- what it is, when to use it, or a link where people can read up about it?